compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/DerivedConfigUtils.cpp

```diff
@@ -181,4 +181,31 @@ SmallVector<int64_t> deriveThreadTileSizes(Operation *op) {
       .Default([&](Operation *op) -> SmallVector<int64_t> { return {}; });
 }
 
+static const int64_t kDefaultGlobalLoadBitSizePerThread =
+    32; // TODO: make it a query.
+SmallVector<int64_t> globalLoadDMATileSizes(Operation *op) {
+  auto funcOp = op->getParentOfType<FunctionOpInterface>();
+
+  std::optional<SmallVector<int64_t>> workgroupSize = getWorkgroupSize(funcOp);
+  if (!workgroupSize) {
+    return {};
+  }
+  [[maybe_unused]] int64_t numThreads =
```

Comment:
This looks always used to me?
```suggestion
  int64_t numThreads =
```